packages/roosterjs-content-model/lib/publicTypes/format/ContentModelFormatMap.ts

```diff
@@ -48,7 +49,7 @@ export interface ContentModelFormatMap {
      */
     tableCell: ContentModelTableCellFormat;
 
-    /**
+    /*
```

Comment:
why change here?

Reply:
It happened when I was solving conflicts. I'll fix.